solr/test-framework/src/java/org/apache/solr/cloud/SolrCloudTestCase.java

```diff
@@ -83,7 +83,7 @@ public class SolrCloudTestCase extends SolrTestCaseJ4 {
 
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  public static final String PRS_DEFAULT_PROP = System.getProperty("use.per-replica", null);
+  public static final String PRS_DEFAULT_PROP = "solr.prs.default";
```

Comment:
Out of scope for now but but I'd like such a setting to exist Solr side officially

Reply:
Put it in Solr where it is used.